### PR TITLE
libseccomp: 2.4.3 -> 2.5.0

### PR DESCRIPTION
**Pull request by saschagrunert**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by saschagrunert** [comment]
cc @thoughtpolice

**Comment by saschagrunert** [comment]
@GrahamcOfBorg build libseccomp

**Comment by saschagrunert** [comment]
@GrahamcOfBorg build libseccomp

**Review by thoughtpolice — CHANGES_REQUESTED** [review]
Please rebase this on top of `staging`, not `master`.

**Comment by saschagrunert** [comment]
> Please rebase this on top of `staging`, not `master`.

Done. Thanks for the hint!

**Review by thoughtpolice — APPROVED** [review]

**Comment by thoughtpolice** [comment]
Thanks!

File: pkgs/development/libraries/libseccomp/default.nix [hunk 1]
```diff
@@ -25,13 +25,13 @@ stdenv.mkDerivation rec {
...
     description = "High level library for the Linux Kernel seccomp filter";
-    homepage    = "https://github.com/seccomp/libseccomp";
-    license     = licenses.lgpl21;
-    platforms   = platforms.linux;
+    homepage = "https://github.com/seccomp/libseccomp";
+    license = licenses.lgpl21;
+    platforms = platforms.linux;
     badPlatforms = [
       "alpha-linux"
-      "riscv64-linux" "riscv32-linux"
```
Comment (JohnAZoidberg) [inline pkgs/development/libraries/libseccomp/default.nix:33]: I only see that they added support for riscv64-linux. (https://github.com/seccomp/libseccomp/blob/5cd9059618a0810ee47c21e6b44c5a876b75e23d/src/arch-riscv64.c) Where do you see riscv32-linux support?

Comment (saschagrunert) [reply pkgs/development/libraries/libseccomp/default.nix:33]: You're right, I readded the architecture there: https://github.com/NixOS/nixpkgs/pull/96337

**Pull request merged**